Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql

```diff
@@ -0,0 +1,139 @@
+--TLWP-1352 - Delete pipeline opportunities for specific users
```

Comment:
Is this file required?

Reply:
File removed - everything is done in the post-deployment script now.